integration-tests/tests/run-workflow-on-snapshot.js

```diff
@@ -66,6 +66,7 @@ const testRunWorkflowOnSnapshotFn = _.flow(
   await launchWorkflowAndWaitForSuccess(page)
 
   await clickNavChildAndLoad(page, 'data')
+  await findElement(page, clickable({ textContains: 'testsnapshot' }));
```

Comment:
`run-workflow-on-snapshot` failed in this CircleCI [job](https://app.circleci.com/pipelines/github/DataBiosphere/terra-ui/10350/workflows/9fc1bd66-fda0-46dd-ab54-eabaf71050ec/jobs/45710/artifacts). Reason: click on "Workspace Data" link did not happen. "OTHER DATA" was clicked instead. This is because clicking happened when `testsnapshot` element was not yet rendered.

<img width="815" alt="Screen Shot 2022-06-27 at 6 19 48 PM" src="https://user-images.githubusercontent.com/35533885/176045445-0799e396-40d5-4fcb-99c9-73ffd0b76137.png">